pandas/tests/indexes/interval/test_interval_range.py

```diff
@@ -29,7 +29,8 @@ class TestIntervalRange:
     @pytest.mark.parametrize("freq, periods", [(1, 100), (2.5, 40), (5, 20), (25, 4)])
     def test_constructor_numeric(self, closed, name, freq, periods):
         start, end = 0, 100
-        breaks = np.arange(101, step=freq)
+        dtype = np.int64 if is_integer(freq) else np.float64
```

Comment:
same as elsewhere, isnt this done automatically now?

Reply:
Yes, after #50195... I'll revert.